lib/adapters/leveldb/leveldb.js

```diff
@@ -1113,7 +1113,7 @@ function LevelPouch(opts, callback) {
         }
         if (++numDone === revs.length) { // done
           if (overallErr) {
-            return callback(err);
+            return callback(overallErr);
```

Comment:
This was clearly a programming error. None of our tests catch it, though, because `compact` never throws an error.